agent/state.go

```diff
@@ -29,23 +29,23 @@ type State struct {
 }
 
 type Info struct {
-	ID      string        `json:"id"`
-	Repo    string        `json:"repository"`
-	Build   string        `json:"build_number"`
-	Started time.Time     `json:"build_started"`
-	Timeout time.Duration `json:"build_timeout"`
+	ID       string        `json:"id"`
```

Comment:
todo migrate json schemas

Reply:
as we do use it in frontend ... should we migrate json schema (fields) already here?

Reply:
#1282